lib/WebpackError.js

```diff
@@ -0,0 +1,21 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Jarid Margolin @jaridmargolin
+*/
+"use strict";
+
+module.exports = class WebpackError extends Error {
+	constructor() {
+		super();
+
+		if(this.initialize) {
+			this.initialize.apply(this, arguments);
```

Comment:
Ahh. Your probably right. Adding an initialize method just creates more tribal knowledge needed to create the subclass.

I'll update the PR to only included the updated inspect method.

I'll then submit a second PR that normalizes the implementations across error classes.